sound/soc/sof/ipc3-topology.c

```diff
@@ -310,13 +310,12 @@ static const struct sof_token_info ipc3_token_list[SOF_TOKEN_COUNT] = {
 static void *sof_comp_alloc(struct snd_sof_widget *swidget, size_t *ipc_size,
 			    int index)
 {
-	u8 nil_uuid[SOF_UUID_SIZE] = {0};
 	struct sof_ipc_comp *comp;
 	size_t total_size = *ipc_size;
 	size_t ext_size = sizeof(swidget->uuid);
 
 	/* only non-zero UUID is valid */
-	if (memcmp(swidget->uuid, nil_uuid, SOF_UUID_SIZE))
+	if (guid_is_null(&swidget->uuid))
```

Comment:
You said `uuid_is_null()` is used in commit message, but in code, `guid_is_null()` is used.

Reply:
again don't pay too much attention for fixup! commit messages @aiChaoSONG, they will disappear when we squash them.